### ISSUE TITLE
Is PSF an RGB image?

### ISSUE THREAD

**Issue by zhouqinhe** [jnjaby/DISCNet#16]
Hi,  I would like to ask, is the PSF calibrated by you in RGB domain or RAW domain? It is said in the paper that ZTE mobile phone is used for direct calibration, so is it RGB?
Best regards!

**Comment by jnjaby** [comment]
Hi, the PSF is calibrated and normalized in RGB domain.

**Comment by zhouqinhe** [comment]
Hi, I do not understand that RGB images appear nonlinear after gamma operation, so how to ensure the linearity in the article? The closed issues [https://github.com/jnjaby/DISCNet/issues/15#issuecomment-1013645741](url) also addresses this problem.
Best regards!

**Comment by jnjaby** [comment]
We did not apply gamma correction on the PSF. Gamma correction is only involved with post-processing on real-captured images.

**Comment by zhouqinhe** [comment]
Maybe my expression is wrong. In Issue#15, I saw that you answered "dump the raw data from camera", because I thought it was calibrated in the raw field, but the training data in the paper was in RGB domain, so I am confused.
Now you say that PSF is calibrated in RGB domain and not Gamma, is this a special RGB/ RAW image? And what is the dynamic range of the PSF?
Looking forward to your reply!

**Comment by jnjaby** [comment]
Oh I see. Processing data in RGB domain does not necessarily mean they are _non-linear_. I suppose you are referring to _sRGB_ gamut where the transfer function is an approximation of gamma curve of 2.2.
In particular, we subtract the black level of raw data, fuse them into an HDR image, and then apply the demosaic process to obtain the _linear RGB_ PSF. In this pipeline, the PSF data remains linear, _i.e._, the pixel values are proportional to the light intensities. As can be found in the data, the HDR PSF is stored with 32-bit floating-point numbers.

**Comment by zhouqinhe** [comment]
Ok, I got it, thanks for your reply!
